src/views/testUtils.js

```diff
@@ -17,7 +17,7 @@ export const visit = path => {
     // From the docs: Note that, even though goto normally waits
     // for all the resources on a page to load, a timeout exception
     // is only raised if the DOM itself has not yet loaded.
-    gotoTimeout: 4000
+    gotoTimeout: process.env.TEST_TIMEOUT || 4000
```

Comment:
And now it won't fail. I tried to record an asciicast of it failing and it processes `ServiceRequest.test.js` in about the same time as the other tests without failing.